test/ffmpeg-qsv/decode/10bit/hevc.py

```diff
@@ -9,6 +9,8 @@
 from ..decoder import DecoderTest
 
 spec = load_test_spec("hevc", "decode", "10bit")
+spec_422 = load_test_spec("hevc", "decode", "10bit", "422p")
+spec_444 = load_test_spec("hevc", "decode", "10bit", "444p")
```

Comment:
What is 422p and 444p?  Doesn't "p" indicate "planar"?  How do these names have any relation to Y210 and Y410?

Reply:
remove this